rezasm-source/rezasm-core/src/instructions/instruction_registry.rs

```diff
@@ -54,6 +54,11 @@ lazy_static! {
     static ref INSTRUCTIONS: Mutex<InstructionRegistry> = Mutex::new(InstructionRegistry::new());
 }
 
+/// Registers an instruction with the static INSTRUCTIONS
+///
+/// # Arguments
```

Comment:
either finish this or get rid of it

Reply:
What do you consider as "finished" documentation? My plan was to at least get one of these basic things down for every important method seeing as the current codebase has all of 3 comments in it. However, I simply do not have the time to write 5 paragraphs 100 times, so this would have to do for now

Reply:
The content of this comment can be inferred from the function name and arguments. I don't think it would be reasonable to ask for you to write a "rust-std" level of documentation ("5 paragraphs"), but I would like indication as to the purpose of the function beyond what can be inferred from the name. For this function I would expect something akin to the following:

```rust
/// Installs the instruction into the global instruction registry. Allows for the given instruction to be accessed
/// anywhere during the program at runtime via the instruction registry.
///
/// # Arguments
///
/// * instruction: a reference to the static-lifetime instruction to register
``` 

I would also like to note that this file was otherwise not touched in this PR. I do not think we should to touch other files' (lack of) documentation in a PR directed at getting the Reader working. The addition of this comment would be better-placed in a PR dedicated to either cleaning up the file (hence also adding documentation) or a PR dedicated to writing documentation on a certain fileset which includes this file.